src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -27,34 +28,51 @@ public class C
 {
     void Method()
     {
-        var a = String.Format("""", 1);
-        var b = String.Format(""{0}"", 1, 2);
-        var c = String.Format(""{0} {1}"", 1, 2, 3);
-        var d = String.Format(""{0} {1} {2}"", 1, 2, 3, 4);
-        var e = string.Format(""{0} {0}"", 1, 2);
-
-        IFormatProvider p = null;
-        var f = String.Format(p, """", 1);
-        var g = String.Format(p, ""{0}"", 1, 2);
-        var h = String.Format(p, ""{0} {1}"", 1, 2, 3);
-        var i = String.Format(p, ""{0} {1} {2}"", 1, 2, 3, 4);
+        {|#0:" + invocation + @"("""", 1)|};
```

Comment:
@jmarolf so on these cases I need to report on all extra args, is it right?

Reply:
I think thats what we want. Once we get the preview out for .NET 6 well see if feedback directs us to change this.

---
In reply to: [508700724](https://github.com/dotnet/roslyn-analyzers/pull/4226#discussion_r508700724) [](ancestors = 508700724)